manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2

```

```suggestion
PackageVersion: 1.9.2
```
This must match Control Panel.